### ISSUE TITLE
Entire eventcard should be clickable

### ISSUE THREAD

**Issue by benaltair** [bahaistudies/conference#30]
Implement with javascript. Accessibility needs to be maintained. Will be complicated potentially if category or speakers are linked, so obvious hover and ARIA data should be added in that case.

For now, `<h1>` will be wrapped in `<a>` tags and a `data-link` attribute or similar will be added to the `<article>` element. Then in JS, this path should be followed on any click that is within the card but not another `<a>` element.

**Comment by benaltair** [comment]
Closing for now, unless we need links within the card. Will have to consider a11y implications later - for instance, will the entire `<article>` card be read by a screen reader every time? Or just the title?
